frontend/src/hooks/usePlausibleTracker.ts

```diff
@@ -69,7 +69,8 @@ export type CustomEvents =
     | 'events-exported'
     | 'event-timeline-open'
     | 'event-timeline-close'
-    | 'event-timeline-event-hover';
+    | 'event-timeline-event-hover'
```

Comment:
Do we need separate event for all of these, we usually use properties?

Reply:
Thanks @sjaanus and @nunogois for the discussion on this, I have changed this back to being based on props